### PR TITLE
Complete setting page

### PR DESCRIPTION
**Pull request by MohammadWasi**
issue Number: #24

### CODE REVIEW

**Comment by MohammadWasi** [comment]
screenshot attached:
settings-mobile
settingsDesktop

File: frontend/src/components/Divider.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Divider from '@material-ui/core/Divider';
+import styled from 'styled-components';
+
+const ZappDivider = styled(Divider)`
+&& { 
+    border: solid 1px;
```
Comment (ahhda) [inline frontend/src/components/Divider.js]: Indentation issue

File: frontend/src/components/Subheading.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const Subheadings = styled.h3`
+border-bottom: 1px solid #000;
```
Comment (ahhda) [inline frontend/src/components/Subheading.js]: Please keep indentations consistent

File: frontend/src/containers/settings/Settings.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        <ItemDetails>
+          <ChangePassword>
+            <StyledButton color="secondary" variant="contained" margin="0"> Change Password</StyledButton>
+          </ChangePassword>
+          <ItemName>
+            <Typography>
+            Primary Email:
+            </Typography>
+            <Typography>
+      dummy-longest-mail@gmail.com
```
Comment (nimish-gupta) [inline frontend/src/containers/settings/Settings.js]: indentation

File: frontend/src/containers/settings/Settings.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            </Typography>
+            <StyledButton color="secondary" variant="contained" margin="0"> Change </StyledButton>
+          </ItemName>
+        </ItemDetails>
+      </ItemContainer>
+
```
Comment (nimish-gupta) [inline frontend/src/containers/settings/Settings.js]: new line?

File: frontend/src/containers/settings/Settings.js [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+      <ZappDivider />
+      <ItemContainer>
+        <Subheadings float="left">Accounts:</Subheadings>
+        <ItemName>
+          <Typography>
+            Slack:
+          </Typography>
+          <Typography>
+      slack-longest-mail@gmail.com
```
Comment (nimish-gupta) [inline frontend/src/containers/settings/Settings.js]: indentation

File: frontend/src/containers/settings/Settings.js [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+          </Typography>
+          <StyledButton color="secondary" variant="contained" margin="0"> Change </StyledButton>
+        </ItemName>
+        <ItemName>
+          <Typography>
+            Slack:
+          </Typography>
+          <Typography>
+      slack-longest-mail@gmail.com
```
Comment (nimish-gupta) [inline frontend/src/containers/settings/Settings.js]: indentation

File: frontend/src/containers/settings/Settings.js [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
```
Comment (nimish-gupta) [inline frontend/src/containers/settings/Settings.js]: indentation.

File: frontend/src/containers/settings/Settings.js [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+          </Typography>
+          <StyledButton color="secondary" variant="contained" margin="0"> Change </StyledButton>
+        </ItemName>
+      </ItemContainer>
+      <ZappDivider />
+      <ItemContainer>
+        <Subheadings float="left">Notifications:</Subheadings>
+        <ItemName>
+          <Typography>
+          Send notification on failed Zapp?
```
Comment (nimish-gupta) [inline frontend/src/containers/settings/Settings.js]: indentation

**Pull request closed without merging**